hector_mapping/CMakeLists.txt

```diff
@@ -1,7 +1,16 @@
 cmake_minimum_required(VERSION 3.0.2)
 project(hector_mapping)
 
-find_package(catkin REQUIRED COMPONENTS roscpp nav_msgs visualization_msgs tf message_filters laser_geometry tf_conversions message_generation)
+find_package(catkin REQUIRED COMPONENTS 
+  roscpp
+  nav_msgs
+  visualization_msgs
+  tf
+  message_filters
+  laser_geometry
+  tf_conversions
+  message_generation
+  std_srvs)
```

Comment:
I separated by one-per-line (for readability), but I can revert this if requested...
In addition, I added the package `std_srvs`  so that I can add the service calls to this code